src/lib/models/date-range.js

```diff
@@ -106,6 +106,16 @@ class DateRange {
     return new DateRange(startDate, endDate);
   }
 
+  /**
+   * Takes an input and turns it into a string representation of a date
+   * in the format YYYY-MM-DD
+   *
+   * @param {*} input A date, moment, string or null to format
+   */
+  static formatDate (input) {
```

Comment:
I'm not certain if this should live here?  Would it be better in src/lib/date or similar?